src/ctap/storage.rs

```diff
@@ -12,16 +12,19 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-use crate::crypto::rng256::Rng256;
 #[cfg(feature = "with_ctap2_1")]
 use crate::ctap::data_formats::{extract_array, extract_text_string};
 use crate::ctap::data_formats::{CredentialProtectionPolicy, PublicKeyCredentialSource};
 use crate::ctap::pin_protocol_v1::PIN_AUTH_LENGTH;
 use crate::ctap::status_code::Ctap2StatusCode;
 use crate::ctap::{key_material, USE_BATCH_ATTESTATION};
+#[cfg(feature = "fuzzing")]
+use crate::embedded_flash::{self, StoreConfig, StoreEntry, StoreError};
```

Comment:
Looks good to me, although I still don't fully understand why the import path changes. But we can discuss splitting away the `embedded_flash` into its own crate as a separate pull request.

Reply:
Resolved by #156.